configure.ac

```diff
@@ -15,6 +15,12 @@ AC_INIT([gluster-block],
 
 AC_SUBST([PACKAGE_RELEASE], 1)
 
+m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES(yes)])
+if make --help 2>&1 | grep -q no-print-directory; then
+    AM_MAKEFLAGS="$AM_MAKEFLAGS --no-print-directory"
+fi
+AC_SUBST(AM_MAKEFLAGS)
+
```

Comment:
@lxbsz I like this change, but with this we cannot see with what all compile/linker flags an individual file is compiled/linked.

I'm okay to keep this as default, as this shows less details to users, but It will be very nice to have an option at configure time to enable/disable this, to help dev at debugging.

Thanks!

Reply:
@pkalever  Make sense and will fix it.

Reply:
@lxbsz 

Can you also check:
$ ./configure --silent
$ make --silent

and see if that is what we are looking at ?

Reply:
Yeah, it is. Just want to make this disabled as default, make sense ?

Reply:
Yup! please provide an option like --disable-silent or something meaningful. Thanks!